src/services/codefixes/convertFunctionToEs6Class.ts

```diff
@@ -34,13 +34,14 @@ namespace ts.codefix {
 
             case SyntaxKind.VariableDeclaration:
                 precedingNode = ctorDeclaration.parent.parent;
+                newClassDeclaration = createClassFromVariableDeclaration(ctorDeclaration as VariableDeclaration);

```

Comment:
I gather this call copies the comments from `ctorDeclaration` so there's no reason to do it below?

Reply:
`createClassFromVariableDeclaration` doesn't do anything about comments, so we still need the call to `copyComments` on line 52 (in addition to the one just below this which copies a different set of comments).